Frameworks/UIKit/UIPanGestureRecognizer.mm

```diff
@@ -541,4 +528,8 @@ + (BOOL)_fireGestures:(id)gestures shouldCancelTouches:(BOOL&)shouldCancelTouche
     return didRecognize;
 }
 
+- (const std::vector<TouchInfo>&)_getTouches {
```

Comment:
copies are wasteful

---
In reply to: [86441503](https://github.com/Microsoft/WinObjC/pull/1305#discussion_r86441503) [](ancestors = 86441503)